DEVELOPMENT.md

```diff
@@ -188,29 +188,58 @@ Use `go test` to run the integration test suite:
 go test -v ./test/integration -tags=integration
 ```
 
+## Running ko builds with other base images
+By default `ko` uses a lightweight image to run go programs.
+The image can be [overridden](https://ko.build/configuration/#overriding-base-images) with
+the `defaultBaseImage` parameter of the `.ko.yaml` configuration file or through the
+environment variable  `KO_DEFAULTBASEIMAGE`.
+
+An example of an image with a set of debugging tools already installed that can be used is
+`registry.redhat.io/rhel9/support-tools`
+
+**[NOTE]**: Using this images usually needs the pod property `runAsNonRoot` set to `false`.
+
 ## Remote IDE debugging
 
 Use [delve](https://golangforall.com/en/post/go-docker-delve-remote-debug.html)
 to start a debugger to which attach from your IDE.
 
 1. Deploy the chart with `ko` and `helm` in debug mode using an image with `delve`:
+   Specify the [component].debug variable to `true` and update the image accordingly.
+
+   * Deployment to debug the API:
```

Comment:
What do you think about having one single command to deploy everything in debug mode? That should simplify the process and we would achieve the same. So it would include

```
export KO_DEFAULTBASEIMAGE=gcr.io/k8s-skaffold/skaffold-debug-support/go:latest
export KO_DISABLE_OPTIMIZATIONS=true

helm installl .... \
  --set global.debug=true \
  ...
```

Reply:
I thought about this but I think that we usually want to debug just one of the components at a time.
Also, I think that this won't be trivial, you need to start one debugger process on the IDE per component and make sure that you use different ports per each.

Another thing that worries me for enabling debugging for all components at the same time is that this process usually consumes much more memory than just running, and that could potentially be an issue

Reply:
Got it!